.storybook/preview-head.html

```diff
@@ -1,10 +1,25 @@
 <link rel="preload" href="./assets/Poppins-Regular.ttf" as="font" type="font/ttf" crossorigin />
+<link rel="preload" href="./assets/Poppins-Semibold.ttf" as="font" type="font/ttf" crossorigin />
+<link rel="preload" href="./assets/Poppins-Bold.ttf" as="font" type="font/ttf" crossorigin />
+
 <style>
   @font-face {
     font-family: 'Poppins';
     src: url('./assets/Poppins-Regular.ttf');
   }
 
+  @font-face {
+    font-family: 'Poppins';
+    src: url('./assets/Poppins-Semibold.ttf');
+    font-weight: 600;
+  }
+
```

Comment:
- The `font-weight: 500;` variant is missing.
- Using a variable font might be a good idea.
- To optimize performance, we should consider converting the TTF files to WOFF2 format.
- Additionally, we might want to take only a subset of the font to further optimize loading times.